code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -305,6 +305,11 @@ LINEN BINS
 	extinguish()
 	update_icon()
 
+/obj/structure/bedsheetbin/wrench_act(mob/user, obj/item/I)
+	if(user.a_intent == INTENT_HARM)
+		. = TRUE
+		default_unfasten_wrench(user, I, time = 20)
```

Comment:
Don't need to set the default return here when you can just return on the next line
```suggestion
		default_unfasten_wrench(user, I, time = 20)
		return TRUE
```